EST.MIT.InvoiceImporter.Function/Models/Notification.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace EST.MIT.InvoiceImporter.Function.Models;
+public enum NotificationType
+{
+    approval,
+    approved,
+    rejected
```

Comment:
Need a few extra notification types in here.
uploaded
upload_fail
upload_success